app/assets/stylesheets/events.scss

```diff
@@ -3,8 +3,80 @@
 // You can use Sass (SCSS) here: http://sass-lang.com/
 
 .simple-calendar{
-  table tbody tr{height:50px;font-size:10px}
-  .day:hover {background-color: lightgrey;}
-  .day.active{background-color: #FFCC99;}
-  .event-cal{margin:0 0 10px 0}
+  table tbody tr {height:50px;font-size:10px;}
+  .day.active {background-color: aliceblue;}
+  .event-cal {margin:0 0 10px 0}
+  .day.prev-month, .day.next-month { background-color: lightgrey; }
+  .day.active.prev-month, .day.active.next-month { background-color: lightgrey; }
+  a { color: #0e6c67; }
+}
+
+@media (max-width: 560px) {
+  .simple-calendar {
+    table tbody {
+      height:50px;
+      font-size:10px;
```

Comment:
It;s best to be consistent with your CSS, in all the code below you have a space after the colon so it's best to stick with that here.